src/wp-includes/class-wp-query.php

```diff
@@ -3555,11 +3584,13 @@ private function set_found_posts( $q, $limits ) {
 			 * Filters the query to run for retrieving the found posts.
 			 *
 			 * @since 2.1.0
+			 * @since x.x.x This query was changed from `SELECT FOUND_ROWS()` to a more
```

Comment:
Just a thought, I wonder if we could make it so we can call `set_found_posts` by itself with the rest of the call. We have examples in core where we get one post to get the count see. 

https://github.com/WordPress/wordpress-develop/blob/722f73761ce60acb52690782dee4782ef5a1a755/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L402